src/components/Todo/Todo.module.css

```diff
@@ -1,49 +1,127 @@
-.wrap {
-    width: 350px;
-    height: 550px;
-    margin: auto;
-    display: flex;
-    flex-flow: column nowrap;
-    align-items: center;
-    justify-content: space-around;
+.todo {
+  position: absolute;
+  width: 700px;
+  height: 644px;
+  margin: auto;
+  background: #ffffff;
+  border-radius: 9px;
+  box-shadow: 0px 0px 3px rgba(0, 0, 0, 0.25);
+}
+
+.wrap_todo {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+}
+
+.todo_header {
+  display: flex;
+  justify-content: space-between;
+  width: 90%;
+  align-items: center;
+  margin: 5%;
 }
 
 .title {
-    font-family: 'Helvetica';
+  font: normal bold 28px Roboto, sans-serif;
+  margin: 0;
 }
 
-.item {
-    color: black;
-    list-style: none;
-    margin-left: -60;
+.item_list {
+  overflow-y: auto;
+  max-height: 432px;
 }
 
 .done {
-    color: #c4c4c4;
-    text-decoration: line-through;
+  color: #c4c4c4;
+  text-decoration: line-through;
 }
 
 .task {
-    margin: auto;
-    margin-left: -40px;
-    width: 250px;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
+  border: 1px solid #cccccc;
+  box-sizing: border-box;
+  border-radius: 41px;
+  width: 636px;
+  height: 36px;
+  margin-bottom: 8px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.task:hover {
+  border: 1px solid #356eff;
+}
+
+.text {
+  cursor: pointer;
+  font: normal normal 14px/16px Roboto, sans-serif;
+    margin-left: 30px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)